recipes/pypy3/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pypy3" %}
+{% set version = "5.10.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://bitbucket.org/pypy/pypy/downloads/pypy3-v{{ version }}-src.tar.bz2
+    fn: pypy3-v{{ version }}-src.tar.bz2
+    sha256: f5548e06e2fc0c24ec8b6e3c5b09f90081818f7caa3e436dc312592611724713
+    folder: pypy3
+    patches:
+      - tklib_build.patch
+
+  - url: https://bitbucket.org/pypy/pypy/downloads/pypy2-v5.9.0-osx64.tar.bz2  # [osx]
```

Comment:
Why is this a different version than the one in the version tag?  You might want some more selectors for that up top so that we don't package this as something it isn't

Reply:
oh right you're using this pypy to do the compile